task/buildah-remote/0.1/buildah-remote.yaml

```diff
@@ -279,7 +279,7 @@ spec:
       with open("./sbom-image.json") as f:
         image_sbom = json.load(f)
 
-      with open("./sbom-source.json") as f:
+      with open("./source/sbom-source.json") as f:
```

Comment:
Are you sure this is right? The syft command above is: `syft dir:$(workspaces.source.path)/source --file=$(workspaces.source.path)/sbom-source.json --output=cyclonedx-json` which means the output file will not be under `./source` I think the current code is correct, as it generates the sbom outside of the users source directory.